selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspaceDetailsTest.java

```diff
@@ -138,13 +138,15 @@ public void workingWithAgents() {
         (name, value) -> {
           Assert.assertEquals(dashboardWorkspace.getAgentState(name), value);
           dashboardWorkspace.switchAgentState(name);
+          loader.waitOnClosed();
```

Comment:
Is it really correct placement on waiting of loader to be closed, so as we had it at the start of **dashboardWorkspace.switchAgentState(name);** method previously in class https://github.com/eclipse/che/pull/6706/commits/95e0809b34d8a9009a0c1678e83d4d3db5ff52aa#diff-b918b4454f3846c11126b5e25fdcb6d5 ?